src/System.CommandLine/Binding/SymbolBindingSide.cs

```diff
@@ -19,7 +19,7 @@ public static SymbolBindingSide Create(Option symbol)
         public static SymbolBindingSide Create(Argument argument)
             => new SymbolBindingSide(argument);
 
-        public ISymbolBase Symbol { get; }
+        public IValueDescriptor Symbol { get; }
```

Comment:
Confused. It thought a ValueDescriptor was a parameter or property, not a symbol.

Reply:
This is a refactoring artifact. This class is going to be removed in this PR in any case.